Rainier-4U-MRW.xml

```diff
@@ -50902,6 +50918,10 @@
 		<id>SYSTEM_VDM_DISABLE</id>
 		<default>0x00</default>
 	</attribute>
+	<attribute>
+		<id>SYS_CLOCK_INTEGRATED_SPARES</id>
```

Comment:
Looks like you accidentally reverted the value for SYS_CLOCK_INTEGRATED_SPARES, needs to be set to 1.

Reply:
Ignore this comment, I thought I was looking at an Everest update.  A blank value is okay for Rainier (though an explicit zero would still be better.